internal/playwright/config.go

```diff
@@ -141,3 +147,29 @@ func SplitSuites(p Project) (Project, Project) {
 
 	return dockerProject, sauceProject
 }
+
+func checkSupportedBrowsers(p *Project) error {
+	supportedBrowsers := map[string]struct{}{
+		"chromium": struct{}{},
+		"firefox":  struct{}{},
+		"webkit":   struct{}{},
+	}
+	supportedBrwsList := []string{"chromium", "firefox", "webkit"}
```

Comment:
Would love to use this opportunity to debunk some myths :D 

> First of all, using maps is one of the most efficient ways to check browsers among supported browsers

It's actually not :) Maps are not inherently fast(er). They are _consistent_ in their lookup time. In fact, a short list has a faster lookup time than a map. To entertain the argument, I coded up a quick benchmark focusing on the _worst case scenarios_ for the list (the value to match is the last one at the end, webkit and the "unsupported browser" usecase).

```
BenchmarkCheckSupportedBrowsersMap
BenchmarkCheckSupportedBrowsersMap/supported_browser
BenchmarkCheckSupportedBrowsersMap/supported_browser-8         	124134030	         9.688 ns/op
BenchmarkCheckSupportedBrowsersMap/unsupported_browser
BenchmarkCheckSupportedBrowsersMap/unsupported_browser-8       	 3413454	       346.7 ns/op
BenchmarkCheckSupportedBrowsersList
BenchmarkCheckSupportedBrowsersList/supported_browser
BenchmarkCheckSupportedBrowsersList/supported_browser-8        	184017376	         7.215 ns/op
BenchmarkCheckSupportedBrowsersList/unsupported_browser
BenchmarkCheckSupportedBrowsersList/unsupported_browser-8      	 3506326	       337.1 ns/op
```

So that's that. Either way though, speed isn't a concern at this location and I'd rather we make the code easier to maintain by keeping only one data structure :)

Reply:
Agreed, my vote is to keep code easier to read and maintain. Milliseconds here won't change the user experience.

Reply:
@alexplischke @vrunoa I agree that for such an amount of data (3 browsers) it doesn't worth supporting 2 data structures and the code is more complicated than it should be.